schema/cassandra/cadence/versioned/v0.12/manifest.json

```diff
@@ -3,6 +3,7 @@
   "MinCompatibleVersion": "0.12",
   "Description": "Support history events v2",
   "SchemaUpdateCqlFiles": [
+    "workflow_order.cql",
```

Comment:
I think 0.12 is already released in Cadence v0.4.0, so you need to create a new versioned change v0.13, please also update description.